Lombiq.Tests.UI/Services/UITestExecutor.cs

```diff
@@ -12,8 +12,8 @@ namespace Lombiq.Tests.UI.Services;
 
 public static class UITestExecutor
 {
-    private static readonly object _initLock = new();
-    private static SemaphoreSlim _semaphoreSlim;
+    private static readonly object _lockObjectOfSemaphoreSlimInitialization = new();
```

Comment:
We know it's an object, no need for Hungarian notation. But this doesn't convey where do we use this lock (there can be any semaphore here).

Reply:
What name do you suggest?